pkgs/by-name/hy/hyprpanel/package.nix

```
@@ -0,0 +1,128 @@
+{
+  lib,
+  config,
+  ags,
+  astal,
+  bluez,
+  bluez-tools,
+  brightnessctl,
+  btop,
+  dart-sass,
+  fetchFromGitHub,
+  glib,
+  gnome-bluetooth,
+  gpu-screen-recorder,
+  gpustat,
+  grimblast,
+  gvfs,
+  hyprpicker,
+  libgtop,
+  libnotify,
+  matugen,
+  networkmanager,
+  python3,
+  pywal,
+  stdenv,
+  swww,
+  upower,
+  wireplumber,
+  wl-clipboard,
+  writeShellScript,
+
+  enableCuda ? config.cudaSupport,
+}:
+ags.bundle {
+  pname = "hyprpanel";
+  version = "unstable-2025-02-17";
```

This is a convention:
```suggestion
  version = "0-unstable-2025-02-17";
```

---

it is `{version}-unstable-{date}` but as hyprpanel has [no releases yet](https://github.com/Jas-SinghFSU/HyprPanel/discussions/792) it is `0`.

---

Yes, I already fixed this